site/app/controllers/grading/ElectronicGraderController.php

```diff
@@ -2833,6 +2833,8 @@ private function getMarkStats(Mark $mark, User $grader) {
         // Show all submitters if grader has permissions, otherwise just section submitters
         $submitter_ids = ($grader->accessFullGrading() ? $all_submitter_ids : $section_submitter_ids);
 
+        $submitter_anon_ids = ($anon != 'unblind') ? $submitter_ids : $submitter_anon_ids = $this->core->getQueries()->getAnonId($submitter_ids);
```

Comment:
Why do you set $submitter_anon_ids again in the last part of the ternary operator? You could remove setting that to this below because the result of the ternary operator will also set $submitter_anon_ids.
```suggestion
        $submitter_anon_ids = ($anon != 'unblind') ? $submitter_ids : $this->core->getQueries()->getAnonId($submitter_ids);
```

Reply:
> Why do you set $submitter_anon_ids again in the last part of the ternary operator? You could remove setting that to this below because the result of the ternary operator will also set $submitter_anon_ids.

Oops, no idea why I did that.